namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-uat/resources/route53.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "laa_crime_apps_team_route53_zone" {
+  name = "uat.court-data-adaptor.service.justice.gov.uk"
```

Comment:
Can you please update the zone name to some thing like
`court-data-adaptor.service.justice.gov.uk` so later you can use sub-domains by naming your ingress as  `uat.court-data-adaptor.service.justice.gov.uk`

Reply:
updated to use a higher level name.